databricks/koalas/tests/test_indexes.py

```diff
@@ -319,3 +319,22 @@ def test_multiindex_from_arrays(self):
         kidx = ks.MultiIndex.from_arrays(arrays)
 
         self.assert_eq(pidx, kidx)
+
+    def test_multiindex_droplevel(self):
+        pidx = pd.MultiIndex.from_tuples([('a', 'x', 1), ('b', 'y', 2)],
+                                         names=['level1', 'level2', 'level3'])
+        kidx = ks.MultiIndex.from_tuples([('a', 'x', 1), ('b', 'y', 2)],
+                                         names=['level1', 'level2', 'level3'])
+        with self.assertRaisesRegex(IndexError,
+                                    "Too many levels: Index has only 3 levels, not 4"):
```

Comment:
according to https://github.com/databricks/koalas/pull/1091/files#r352351964,

i think this should be better changed to `"Too many levels: Index has only 3 levels, not 5"`